### DISCUSSION TITLE
Merge Functionality in onpremise Console

### DISCUSSION THREAD

**Discussion by Noste92** [mia-platform/community#330] [Q&A] [answered]
### Which section?

Console

### Which is the version?

11.7.1

### Description

Hi,

I noticed that the merge feature isn't available in our on-premise Console. 
Any idea why? Do we need to enable certain toggles, or could it be related to using Github as our Git provider?

Thank you,
Stefano

**Comment by fredmaggiowski** [comment]
Hi @Noste92 

I think that it could have been manually deactivated, can you access the Console CMS and verify the feature activation for `ENABLE_MERGE_CONFIGURATION` is present and published?

**Reply by Noste92** [reply]
Hi @fredmaggiowski,

I've tried to check on the Console CMS and it seems active, here's the screen:

![image](https://github.com/mia-platform/community/assets/14860750/1796dd43-c707-443c-8c65-7f1a1e1234c1)

Could I check anything else?

**Reply by fredmaggiowski** [reply]
inspecting the Console network do you see feature toggle apis going fine or throwing an error? 🤔

**Reply by Noste92** [reply]
The api calls seems fine, but it doesn't get ENABLE_MERGE_CONFIGURATION feature toogle.
![image](https://github.com/mia-platform/community/assets/14860750/467ac7ca-af90-4935-9190-ca43c237aa9e)
if a tried a similiar call in the PaaS i can see the ENABLE_MERGE_CONFIGURATION feature toggle.

I've also tried on premise to call directly the feature toogle ENABLE_MERGE_CONFIGURATION and the result it's true

![image](https://github.com/mia-platform/community/assets/14860750/967fa05d-ee6b-48f7-aeb0-bbf8ed9dc65b)

**Reply by fredmaggiowski** [reply]
you won't find the `ENABLE_MERGE_CONFIGURATION` in the first API you provided here, you can only get it with the second API you posted `/api/feature-toggles/ENABLE_MERGE_CONFIGURATION`. Does the frontend asks for this feature toggle? Do you find it in the Console network tab when using the app? 🤔

**Reply by Noste92** [reply]
Unfortunatly I cannot find any call to this API `/api/feature-toggles/ENABLE_MERGE_CONFIGURATION`, I don't know what I am missing.

![image](https://github.com/mia-platform/community/assets/14860750/97f97340-1b3e-4abe-980c-9ba8a2bf9829)

**Reply by fredmaggiowski** [reply]
so you don't even see this icon, right?

<img width="289" alt="Screenshot 2023-10-24 alle 16 01 28" src="https://github.com/mia-platform/community/assets/7142570/13d21754-5656-4325-af17-1f3e6b50f812">

**Reply by Noste92** [reply]
Yes, exactly
![image](https://github.com/mia-platform/community/assets/14860750/7ee3a4fe-fafc-48b1-b224-3d7852de6af1)

**Reply by fredmaggiowski** [reply]
ok, can you open a ticket on the internal board so that we can further investigate the issue, with the information you provided I honestly struggle to find what the problem might be 🤔

**Reply by fredmaggiowski** [reply, marked as answer]
Hey @Noste92 while talking with @MIA-Deltat1995 we noticed that the merge feature is not controlled by the database yet as it is controlled via the helm installation chart

Setting the `enableMergeConfiguration: true` in your Helm value file should fix the issue!

**Reply by giulioroggero** [reply]
@Noste92 the solution proposed by @fredmaggiowski solve your problem?

**Reply by Noste92** [reply]
Hi @fredmaggiowski @Noste92, we are looking for the right spot to update the Console. I'll update you as soon as possibile.
